src/composables/use-media-query.js

```diff
@@ -13,7 +13,7 @@ export function useMediaQuery(query, options = {}) {
   if (!window) return ref(false)
 
   const mediaQuery = window.matchMedia(query)
-  /** @type {import('@nuxtjs/composition-api').Ref<boolean>} */
+  /** @type {import('@nuxtjs/composition-api').Ref<Boolean>} */
```

Comment:
`Boolean` with an uppercase b actually refers to the primitive constructor literally rather than the abstract type for `boolean`. Same with the other JS primitives. Object is even more complicated as there are _three_ ways to reference an object type: https://stackoverflow.com/questions/49464634/difference-between-object-and-object-in-typescript

```suggestion
  /** @type {import('@nuxtjs/composition-api').Ref<boolean>} */
```

Reply:
Done

Reply:
This seems to have gone back to the original state.